src/12-nextgen-projects/projectLines.apib

```diff
@@ -0,0 +1,63 @@
+## Project lines [/nextgen/projectLines]
+
+> A project line can be a group, a task, or a material.
+> The `projectLines.list` endpoint only returns `{id, type}` pairs of those lines. If you want more information, you should "side-load" those entities with `?include=line`.
```

Comment:
The documentation mentions [sideloading](https://developer.teamleader.eu/#/introduction/sideloading) instead of 'side-loading'. It also explains how it works so not sure if we should explicitly mention how to do it here.